### ISSUE TITLE
MergeableSelector should allow a whitelist

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#540]
For anyone using Polymer (and some versions of Angular), there are a few bizarre CSS rules that you can sprinkle into your CSS that do magical things [1] [2]. Here's an example from Polymer's docs:

```css
polyfill-next-selector { content: ':host > *' }
::content > * { }

polyfill-rule {
  content: '.bar';
  background: red;
}

polyfill-rule {
  content: ':host.foo .bar';
  background: blue;
}
```

I know, it's bizarre. Because of how these "rules" work, you will repeat them multiple times throughout your CSS, and MergeableSelector will report on them. Can MergeableSelector support a whitelist of selectors that the developer intends to repeat, because a JS library does magic with it? I would use it like:

```yaml
MergeableSelector:
  whitelist: ['polyfill-next-selector', 'polyfill-rule', 'polyfill-unscoped-rule']
```

[1] https://www.polymer-project.org/0.5/docs/polymer/styling.html#at-polyfill
[2] https://github.com/angular/angular.dart/wiki/CSS-Shim
